fighters/lucas/src/acmd/specials.rs

```diff
@@ -238,59 +230,182 @@ unsafe fn lucas_special_lw_end_effect(fighter: &mut L2CAgentBase) {
+        ATTACK(fighter, 0, 0, Hash40::new("top"), 0.0, 10, 0, 0, 55, 14.0, 0.0, 10.0, 0.0, None, None, None, 0.0, 1.0, *ATTACK_SETOFF_KIND_OFF, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, true, true, false, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_NONE, *ATTACK_REGION_NONE);
     }
-    wait(lua_state, 1.0);
+    wait(lua_state, 3.0);
     if is_excute(fighter) {
         AttackModule::clear_all(boma);
-        FT_MOTION_RATE(fighter, 0.800);
     }
-    
 }
 
-#[acmd_script( agent = "lucas", script = "effect_specialairlwend" , category = ACMD_EFFECT , low_priority)]
-unsafe fn lucas_special_air_lw_end_effect(fighter: &mut L2CAgentBase) {
+#[acmd_script (agent = "lucas", scripts = ["sound_specialnstart", "sound_specialairnstart"], category = ACMD_SOUND, low_priority)]
+unsafe fn lucas_special_n_start_sound(fighter: &mut L2CAgentBase) {
+    let lua_state = fighter.lua_state_agent;
+    let boma = fighter.boma();
+    frame(lua_state, 13.0);
+    if is_excute(fighter) {
+        PLAY_SE_REMAIN(fighter, Hash40::new("se_lucas_smash_l03"));
+        PLAY_SE_REMAIN(fighter, Hash40::new("vc_lucas_attack06"));
+    }
+}
+
+// SPECIAL N HOLD //
+
+#[acmd_script ( agent = "lucas", scripts = ["game_specialnhold", "game_specialairnhold"], category = ACMD_GAME, low_priority)]
+unsafe fn lucas_special_n_hold(fighter: &mut L2CAgentBase) {
+    // INTENTIONALLY LEFT BLANK
+}
+
+#[acmd_script (agent = "lucas", scripts = ["effect_specialnhold", "effect_specialairnhold"], category = ACMD_EFFECT, low_priority)]
+unsafe fn lucas_special_n_hold_effect(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     frame(lua_state, 1.0);
     if is_excute(fighter) {
-        EFFECT_FOLLOW(fighter, Hash40::new("lucas_psimagnet_end"), Hash40::new("trans"), 0, 6.5, 12, 0, 0, 0, 0.4, true);
-        FLASH(fighter, 0.5, 1, 1, 0.4);
-        EFFECT_DETACH_KIND(fighter, Hash40::new("lucas_psimagnet_end"), -1);
+        FOOT_EFFECT(fighter, Hash40::new("null"), Hash40::new("top"), 0, 0, 0, 0, 0, 0, 1, 9, 0, 9, 0, 0, 0, false);
+        FLASH(fighter, 0.01, 0.5, 1, 0.4);
     }
-    wait(lua_state, 5.0);
+    for i in 1..=50 {
+        if is_excute(fighter) {
+            if i%2==0 {
+                EFFECT_OFF_KIND(fighter, Hash40::new("lucas_pkfr_hold"), false, false);
+                EFFECT_FLW_POS(fighter, Hash40::new("lucas_pkfr_hold"), Hash40::new("top"), 0, sv_math::rand(hash40("fighter"), 4) as i32 + 12, sv_math::rand(hash40("fighter"), 4) as i32 - 2, 0, 0, 0, 0.5, true);
```

Comment:
You can use the `emit` bone to make an effect spawn randomly on a character's body